update-compose.yaml

```diff
@@ -0,0 +1,12 @@
+policies:
+  - name: Handle apm-data server specs
```

Comment:
Unfortunately these "name:"  `Handle ...` strings do not appear in the workflow run log at all, e.g.: https://github.com/elastic/apm-agent-nodejs/actions/runs/9806099782/job/27077125378

Reply:
I'll raise an issue in the upstream project so it can be honoured or at least they know this case. Thanks